packages/toolpad-components/src/Chart.tsx

```diff
@@ -205,7 +204,8 @@ export default createBuiltin(Chart, {
   loadingProp: 'loading',
   loadingPropSource: ['data'],
   errorProp: 'error',
-  resizableHeightProp: 'height',
+  defaultLayoutHeight: 360,
```

Comment:
New interface that should be simpler.
Any component is vertically resizable, but we can set a default and minimum height for specific components.